Block/Tealium.php

```diff
@@ -209,33 +228,65 @@ public function render($type = null, $external = false, $sync = "sync")
             } else {
                 $udoJson = "{}";
             }
-/* echo "test<pre>";
+            /* echo "test<pre>";
 			print_r($udoObject);
 			die; */
             // create the javascript for utag_data
             $udoJs = "var utag_data = $udoJson;";
 
-            // create the entire script tag to render for utag_data
-            $udo = <<<EOD
-<!-- Tealium Universal Data Object / Data Layer -->
-<div class="utagLib" style="display:none;">//tags.tiqcdn.com/utag/$this->account/$this->profile/$this->target/utag.js</div>
-<script type="text/javascript">
-$udoJs
-console.log(window);
-</script>
-<!-- ****************************************** -->
-EOD;
+            
+            $this->fpurl = trim($this->fpurl);
+            if (!empty($this->fpurl) && $this->fpurl != "") {
+                // create the entire script tag to render for utag_data
+                $udo = <<<EOD
+                <!-- Tealium Universal Data Object / Data Layer -->
+                <div class="utagLib" style="display:none;">https://$this->fpurl/$this->profile/$this->target/utag.js</div>

```

Comment:
There must be a better way of storing the loaded URL than including a hidden div?

Reply:
Changed to JS var